object/object.go

```diff
@@ -350,6 +353,25 @@ func (o *Object) Attributes() []Attribute {
 	return res
 }
 
+// NonSystemAttributes returns non-system object attributes.
+//
+// See also SetAttributes, Attributes.
+func (o *Object) NonSystemAttributes() []Attribute {
```

Comment:
We have a notion of "user attributes" in the spec, maybe it'd be better than "non-system attributes"?

Reply:
thought about that. [This](https://github.com/nspcc-dev/neofs-api/blob/7ae1a7438b19fc4ca32c451db67de170edb539a7/object/types.proto#L156-L157) bothers me a bit. Mb u mentioned some other spec piece?

Reply:
More like these
https://github.com/nspcc-dev/neofs-spec/blob/master/01-arch/01-netmap.md#network-map
https://github.com/nspcc-dev/neofs-spec/blob/master/01-arch/07-acl.md#extended-acl

Reply:
name `User`